From: Rylan Dmello <mail@rylan.coffee>
Date: Thu, 23 Apr 2020 22:57:23 -0400
Subject: staging: qlge: qlge_dbg.c: Remove trailing semicolon from macro
Patch-mainline: v5.8-rc1
Git-commit: f80fa161ae616e648f7040544c0e1f5f225fbfbf
References: jsc#SLE-15139

Fix checkpatch.pl warning: macros should not use a trailing semicolon

Signed-off-by: Rylan Dmello <mail@rylan.coffee>
Link: https://lore.kernel.org/r/20200424025723.GA28156@athena
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Shung-Hsi Yu <shung-hsi.yu@suse.com>
---
 drivers/staging/qlge/qlge_dbg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/qlge/qlge_dbg.c
+++ b/drivers/staging/qlge/qlge_dbg.c
@@ -1564,7 +1564,7 @@ void ql_dump_stat(struct ql_adapter *qde
 	pr_err("qdev->%-24s = %llx\n", #field, (unsigned long long)qdev->field)
 #define DUMP_QDEV_ARRAY(qdev, type, array, index, field) \
 	pr_err("%s[%d].%s = " type "\n",		 \
-	       #array, index, #field, (qdev)->array[index].field);
+	       #array, index, #field, (qdev)->array[index].field)
 void ql_dump_qdev(struct ql_adapter *qdev)
 {
 	int i;
